docs/cluster-setup-guide/deploy-cluster/sysadmin-deploy-on-k8s/overview.rst

```diff
@@ -61,6 +61,14 @@ the `Kubernetes Cluster Autoscaler
 `GKE <https://cloud.google.com/kubernetes-engine/docs/concepts/cluster-autoscaler>`_ and `EKS
 <https://docs.aws.amazon.com/eks/latest/userguide/cluster-autoscaler.html>`_.
 
+Pod Security
+============
+
+Determined runs task containers as root by default. However, one can link a determined user to a
```

Comment:
Is there guidance on whether we use 2nd or 3rd person subjects in the docs? `One` feels a bit weird to say if this is a guide.